backend/adapter_processor/serializers.py

```diff
@@ -94,6 +94,40 @@ def to_representation(self, instance: AdapterInstance) -> dict[str, str]:
         return rep
 
 
+class AdapterInfoSerializer(BaseAdapterSerializer):
+
+    context_window_size = serializers.SerializerMethodField()
+
+    class Meta(BaseAdapterSerializer.Meta):
+        model = AdapterInstance
+        fields = (
+            "id",
+            "adapter_id",
+            "adapter_name",
+            "adapter_type",
+            "created_by",
+            "context_window_size",
+        )  # type: ignore
+
+    def get_context_window_size(self, obj: AdapterInstance) -> Any:
```

Comment:
@johnyrahul any way to re-use the existing context-window size function that we have in the get adapter instance API handler? Or could that re-use this method?

Reply:
Other parts look fine.

Reply:
@gaya3-zipstack  do we require that function in the get adapter instance function anymore?
I wanted to check with you before removing them.

Reply:
@johnyrahul In my opinion, it will be a good practice for the details API to return all relevant information. This is from the design principle and the maintenance perspective. We can always call the GET adapter instance API to get the superset information.

Reply:
addressed